polkapox/1.0.0-beta/README.md

```diff
@@ -0,0 +1,16 @@
+# PolkaPox container
+
+Main tool: [polkapox](https://github.com/CDCgov/polkapox)
+  
+Additional tools installed via pip:
+- biopython: 1.83
+- numpy: 1.24.4
+- mkl: 2024.0.0
+- networkx: 3.1
+- gfapy: 1.2.3
+Additional tools installed from pre-compiled binaries:
+- Blast+: 2.15.0
+
+[polkapox](https://github.com/CDCgov/polkapox) is a nextflow workflow for filtering, trimming, QC, reference-based analysis, and de novo assembly of Illumina sequencing reads from orthopoxviruses. The resultant container is not intended to run independently of the workflow.
+
+Full documentation: [https://github.com/CDCgov/polkapox](https://github.com/CDCgov/polkapox)
```

Comment:
I have no changes to recommend on the README.md. 

If I'm being extra picky, I would list the main tools as the python packages.